src/app/GitUI/UserControls/FileStatusList.cs

```diff
@@ -379,7 +379,10 @@ private void SetFindInCommitFilesGitGrepVisibilityImpl(bool visible)
             cboFindInCommitFilesGitGrep.Visible = visible;
             if (visible)
             {
-                ActiveControl = cboFindInCommitFilesGitGrep;
+                if (ActiveControl is not null)
+                {
+                    ActiveControl = cboFindInCommitFilesGitGrep;
+                }
```

Comment:
The new code is rather confusing: if the active control is set - why change? If it's indeed intentional, then this block of code deserves a comment.